src/gnmi/proto/py.typed

```diff
@@ -0,0 +1 @@
+Marker
```

Comment:
This file can be empty right?

Reply:
Yep, I just copied [aiohttp](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/py.typed) but can definitely make it empty instead.